src/stakingSetup.js

```diff
@@ -0,0 +1,153 @@
+const Keyring = require('@polkadot/keyring').default;
+const testingPairs = require('@plugnet/keyring/testingPairs');
+let log = require('console-log-level')({ level: 'info' });
+const DEFAULT_SLEEP = 5000;
+const keyring = new Keyring({ type: 'sr25519' });
+const defaulKeyrings = testingPairs.default({ type: 'sr25519' });
+
+
+const controllersKeyring = [
+    defaulKeyrings.bob,
+    defaulKeyrings.charlie,
+    defaulKeyrings.eve
+]; 
+
+const controllerAddrs = [
+    "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty", // BOB
+    "5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y", // CHARLIE
+    "5HGjWAeFDfFCWPsjFQdVV2Msvz2XtMktvgocEZcCj68kUMaw" // EVE
+]
+
+async function createStashAccounts(api) {
+    stashAccount = []
+    stashNames = ["BOB_STASH", "CHARLIE_STASH", "EVE_STASH"];
+    acliceKeyring = defaulKeyrings.alice;
+
+    for (i = 0; i < stashNames.length; i++) {
+        name = stashNames[i];
+        stashAccount.push(keyring.addFromUri(name, { name: name }));
+    }
+
+    for (account of stashAccount) {
+        log.info("SETTING UP FOR CENNZ -------------- \n Accout raw seed: " + account.meta.name)
+        log.info("Transfering cennz to " + account.meta.name)            
+            const cennz = await api.tx.genericAsset.transfer(16000, account.address, "100_000_000_000_000")
```

Comment:
This should not be named cennz.
In this case,  await api.tx.genericAsser.transfer returns a callback which must be executed to terminate continuous async calls.
This is called a subscription in the polkadot docs: https://polkadot.js.org/api/start/api.query.subs.html so we should call the value 'unsub' not 'cennz'